app/controllers/admin/reports/registered_adviser_controller.rb

```diff
@@ -0,0 +1,30 @@
+# require 'csv'
+
+# class AdviserCSVList
+#  def initialize(advisers)
+#    @advisers = advisers
+#  end
+
+#  def self.to_csv(_ = {})
+#    CSV.generate do |csv|
+#      csv << ['Ref. Number', 'Name', 'Firm']
+
+#      @advisers.each do |adviser|
+#        csv << [adviser.reference_number, adviser.name, adviser.firm.registered_name]
+#      end
+#    end
+#  end
+# end
+
+module Admin
+  module Reports
+    class RegisteredAdviserController < Admin::ApplicationController
+      respond_to :html, :csv
+
+      def index
+        @registered_advisers = Adviser.all.page(params[:page])
+        respond_with(@registered_advisers)
```

Comment:
It looks to me that `@registered_advisers` is a paginated number of the total. This is fine for the html page, *but* does the csv then only return the same results rather than ALL of the registered advisors? 
Is this the requirements we need and are there any tests to cover this count?